salt/core/network/files/networkd_linode.template

```diff
@@ -0,0 +1,20 @@
+[Match]
+Name=eth0
+
+[Network]
+DHCP=no
+DNS=178.79.182.5 176.58.107.5 176.58.116.5 176.58.121.5 151.236.220.5 212.71.252.5 212.71.253.5 109.74.192.20 109.74.193.20 109.74.194.20 2a01:7e00::9 2a01:7e00::3 2a01:7e00::c 2a01:7e00::5 2a01:7e00::6 2a01:7e00::8 2a01:7e00::b 2a01:7e00::4 2a01:7e00::7 2a01:7e00::2
+Domains=open-contracting.org
+IPv6PrivacyExtensions=false
+IPv6AcceptRA=false
+
+Address={{ pillar.network.ipv4 }}/24
+{%- if 'ipv6' in pillar.network %}
+Address={{ pillar.network.ipv6 }}/64
+{%- endif %}
+{%- for address in pillar.network.networkd.get('addresses', []) %}
+Address={{ address }}
+{%- endfor %}
```

Comment:
In the netplan template, there's a comment about the last IP being used as the primary IP, and the ipv6 is listed last. Should we do the same here, or does the same behavior not occur?

Reply:
From my testing the order doesn't seem to matter. I'm not sure how it chooses which IP to use for outbound traffic.

I don't expect this to be a problem unless we wanted the SLAAC IP configured on the server as well as the /64 block. I can't think of a use-case when we would want this so, we can investigate more when needed.

Reply:
Ok, sounds good!